src/app/services/profile/profile.service.ts

```diff
@@ -0,0 +1,40 @@
+import { InviteMemberDtoResponse } from 'src/app/shared/models/DtoResponse/invite-member.model';
+import { CreateGroupDtoRequest } from './../../shared/models/DtoRequest/create-group.model';
+import { GroupDtoResponse } from './../../shared/models/DtoResponse/group.model';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { InviteMemberDtoRequest } from 'src/app/shared/models/DtoRequest/invite-member.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService {
+
+  private urlGetGroup = 'http://127.0.0.1:4040/api/group';
```

Comment:
N'oublie pas d'utiliser les variables d'environnements (cet url peut changer)
```suggestion
  private urlGetGroup = environment.api_url + '/group';
```